src/Server.php

```diff
@@ -88,10 +88,6 @@ public function createConnection($socket)
         } else {
             $connection = new SecureConnection($socket, $this->loop);
             $connection->setProtocol($this->getSecureProtocolNumber($context));
-            $scope = $this;
-            $connection->on('connection', function ($dataConn) use ($scope) {
-                $scope->emit('connection', array($dataConn));
-            });
```

Comment:
It seems to be duplicated to https://github.com/boite/socket/blob/develop/src/SecureConnection.php#L27

```php
if (! $this->isSecure) {
    $enabled = stream_socket_enable_crypto($stream, true, $this->protocolNumber);
    if ($enabled === false) {
        $this
            ->err('Failed to complete a secure handshake with the client.')
            ->end()
        ;
        return;
    } elseif ($enabled === 0) {
        return;
    }
    $this->isSecure = true;
    $this->emit('connection', array($this));
}
```

Actually I verified the duplicated issue in my product [mpyw/php-hyper-builtin-server](https://github.com/mpyw/php-hyper-builtin-server) and the my commit fix the issue.